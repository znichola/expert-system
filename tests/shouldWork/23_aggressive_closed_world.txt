A+B=>C
C+T<=>R
=ATR
?CR
# an aggressive closed world would lead to a contradiction in the ruleset