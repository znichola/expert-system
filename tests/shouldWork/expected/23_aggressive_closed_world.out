C is True
R is True
